lib/supercluster.dart

```diff
@@ -1,5 +1,3 @@
-library supercluster;
-
```

Comment:
How come the library declaration was removed?

Reply:
Good catch! It was accidental while I was testing and forgot to put it back.

Reply:
I amended the commit to add back the library directive.